service-deployment/paiLibrary/clusterObjectModel/objectModelFactory.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation
```

Comment:
Why both of kubernetes-deployment and service-deployment have the same 'clusterObjectModel' folder?

Reply:
Because we have no time to integrate them into one module before this release.